kube-up.sh

```diff
@@ -67,15 +74,14 @@ docker run \
     --volume=/sys:/sys:ro \
     --volume=/var/lib/docker/:/var/lib/docker:rw \
     --volume=/var/run:/var/run:rw \
-    --volume=/var/lib/kubelet:/var/lib/kubelet:rw \
+    --volume=/var/lib/kubelet:/var/lib/kubelet:shared \
     $private_repo_creds_mount \
     --net=host \
     --pid=host \
     --privileged=true \
     -d \
     gcr.io/google_containers/hyperkube-amd64:v1.2.0 \
     /hyperkube kubelet \
-        --containerized \
```

Comment:
containerized was they way they did nsenter's (which they consider an abomination) to emulate the thing that is now being down through mounting as shared above.